wire-library/wire-compiler/src/main/java/com/squareup/wire/schema/Target.kt

```diff
@@ -77,20 +81,42 @@ sealed class Target : Serializable {
       protoFile: ProtoFile,
       emittingRules: EmittingRules,
       claimedDefinitions: ClaimedDefinitions,
+      claimedTypePaths: MutableMap<Path, Type>,
```

Comment:
single map so they can’t mutually conflict?

Reply:
I am not sure it makes sense but I'll try anyway. `MutableMap<Path, Any>` would be gross though.

Reply:
fixed! not so gross afterall